cli/crates/common/src/environment.rs

```diff
@@ -195,20 +192,15 @@ impl Project {
     fn try_init(warnings: &mut Vec<Warning>) -> Result<Self, CommonError> {
         let schema_path = get_project_grafbase_path(warnings)?.ok_or(CommonError::FindGrafbaseDirectory)?;
 
-        let grafbase_directory_path = schema_path
-            .parent()
-            .expect("the schema directory must have a parent by definiton")
-            .to_path_buf();
-
-        let path = grafbase_directory_path
+        let path = schema_path
             .parent()
             .expect("the grafbase directory must have a parent directory by definition")
```

Comment:
Do we always expect a parent here? It seems unlikely that users would have their project at the filesystem root, but it's not unthinkable.

Reply:
`schema_path` would then be `/schema.graphql` or `/grafbase.config.ts`, so `parent()` for those is '/'.

Reply:
Ah yes, good point, all good then :+1: